AccountFlag enum

bartekn opened the pull request:
This PR introduces `AccountFlag` enum that exposes account flags.

Close https://github.com/stellar/java-stellar-sdk/issues/31

nullstyle commented:
+1

Mr0grog commented:
Do the docs need to be regenerated for this? I'm not seeing it at https://stellar.github.io/java-stellar-sdk/

bartekn commented:
@Mr0grog thanks for catching this. Updated the docs just now and also added it to the build process in https://github.com/stellar/java-stellar-sdk/commit/b35d4e5ecd26e73de7feae7e30124a572e92ec15.
